modules/sshkey/versions.tf

```diff
@@ -2,7 +2,11 @@ terraform {
   required_providers {
     equinix = {
       source = "equinix/equinix"
+      version = "1.14.1"
     }
   }
-  required_version = ">= 0.13"
+  required_version = ">= 1.0.0"
+  provider_meta "equinix" {
+    module_name = "equinix-metal-openshift-on-baremetal sshkey"
```

Comment:
I think we've used `/` as the separator between module names elsewhere.  If that's the case, we should to that in these modules as well.